server/src/routes/api/state.mjs

```diff
@@ -289,11 +289,44 @@ function revertState(req, res) {
   });
 }
 
+// POST /api/v1/state/interactionService
+// Expected body: { enabled: true|false, user: 'xxx' }
+// where user is the user ID the interaction logs to be sent
+//  E.g., { enabled: true, user: '456~A' }
+function interactionService(req, res) {
```

Comment:
See comment at top about moving this to user.mjs and separating into two different functions.

Reply:
done